src/common/websocket/api.ts

```diff
@@ -54,6 +54,7 @@ export default class DerivWS {
         const connection = new WebSocket(this.socket_url)
         connection.onopen = this.openHandler
         connection.onmessage = this.messageHandler
+        connection.onclose = this.reloadPage
```

Comment:
Reloading the page should not be preferred. Can we try to re-establish the WS connection?

Reply:
Yes @balakrishna-binary actually I'm trying to do it

Reply:
Done @balakrishna-binary please check one more time 🙏